detail/detail.js

```diff
@@ -3,7 +3,10 @@ import { renderDogDetail } from '../render-utils.js';
 
 const dogDetailContainer = document.getElementById('dog-detail-container');
 
-// on load
-// get the id from URL
```

Comment:
Nice job! When the window loads this event listener will load which grabs the param by id to render a single dog from the supabase table.